packages/styles/scss/components/tabs/_tabs.scss

```diff
@@ -831,13 +442,6 @@
       color: $text-on-color-disabled;
     }
 
-    //-----------------------------
-    //  Tab Content Container
-    //-----------------------------
-    .#{$prefix}--tab-content {
-      padding: $spacing-05;
-    }
-
```

Comment:
Just as an FYI to reviewers, `tab-content` is still a relevant class name but these styles can be removed because they are a duplicate.